### ISSUE TITLE
Newest Update interferes/breaks Shortcodes Ultimate plugin

### ISSUE THREAD

**Issue by extremecarver** [stcr/subscribe-to-comments-reloaded#702]
The newest update somehow interferes with Shortcodes Ultimate - most of the shortcodes (e.g. tabs) are not working anymore after updating Subscribe to Comments reloaded plugin.

It surely happened due to the auto update of the Subscribe to Comments plugin - and deactivating Subscribe to Comments fixed it. However strangely after I reactivated Subscribe to Comments again I cannot replicate the error.

**Comment by BobaWebDev** [comment]
Hi @extremecarver 

The issue is no longer happening after you reactivated StCR?

**Comment by extremecarver** [comment]
yes very strange - the error was there for sure (two websites) after the auto update - but a single time deactivating then reactivating StCR fixed it. I played around with cache and so on but was unable to get the error back or look into the console about the error message (no other plugins auto updated or so in the meantime).
